src/Management/src/Endpoint/Actuators/HttpExchanges/HttpExchange.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Xml;
+
+namespace Steeltoe.Management.Endpoint.Actuators.HttpExchanges;
+
+public sealed class HttpExchange
+{
+    public DateTime Timestamp { get; }
+    public HttpExchangePrincipal? Principal { get; }
+    public HttpExchangeSession? Session { get; }
+    public HttpExchangeRequest Request { get; }
+    public HttpExchangeResponse Response { get; }
+    public string TimeTaken { get; }
```

Comment:
I'd prefer if this were a `TimeSpan`, so it can be sorted on or used in custom filter logic. The formatting should be achieved by using `[JsonIgnore]` and adding an extra property with `[JsonPropertyName]` that calls `XmlConvert.ToString`. Then we don't need two constructors anymore.